arrows/klv/klv_convert_vital.cxx

```diff
@@ -241,19 +241,33 @@ klv_0104_to_vital_metadata( klv_timeline const& klv_data, uint64_t timestamp,
     klv_data.at( standard, KLV_0104_START_DATETIME, timestamp );
   if( start_datetime.valid() )
   {
-    auto const value = start_datetime.get< std::string >();
-    vital_data.add< kv::VITAL_META_START_TIMESTAMP >(
-        kv::std_0104_datetime_to_unix_timestamp( value ) );
+    try
+    {
+      auto const value = start_datetime.get< std::string >();
+      vital_data.add< kv::VITAL_META_START_TIMESTAMP >(
+          kv::std_0104_datetime_to_unix_timestamp( value ) );
+    }
+    catch( kv::metadata_exception const& e )
+    {
+      LOG_ERROR( kv::get_logger( "klv" ), e.what() );
+    }
   }
 
   // Parse the datetime string into a UNIX microsecond timestamp
   auto const event_start_datetime =
-    klv_data.at( standard, KLV_0104_START_DATETIME, timestamp );
+    klv_data.at( standard, KLV_0104_EVENT_START_DATETIME, timestamp );
   if( start_datetime.valid() )
   {
-    auto const value = event_start_datetime.get< std::string >();
-    vital_data.add< kv::VITAL_META_EVENT_START_TIMESTAMP >(
-        kv::std_0104_datetime_to_unix_timestamp( value ) );
+    try
+    {
+      auto const value = event_start_datetime.get< std::string >();
+      vital_data.add< kv::VITAL_META_EVENT_START_TIMESTAMP >(
+          kv::std_0104_datetime_to_unix_timestamp( value ) );
+    }
+    catch( kv::metadata_exception const& e )
+    {
+      LOG_ERROR( kv::get_logger( "klv" ), e.what() );
+    }
   }
```

Comment:
This seems repetitive with the if statement above - possibly make into a function?